src/coreimage.cs

```diff
@@ -5673,85 +5498,3365 @@ interface CIKeystoneCorrection {
 		[CoreImageFilterProperty ("inputFocalLength")]
 		float FocalLength { get; set; }
 
+#if !XAMCORE_4_0
+		[Obsolete ("Use 'InputTopRight' instead.")]
 		[CoreImageFilterProperty ("inputTopRight")]
 		CIVector TopRight { get; set; }
 
+		[Obsolete ("Use 'InputBottomRight' instead.")]
 		[CoreImageFilterProperty ("inputBottomRight")]
 		CIVector BottomRight { get; set; }
 
+		[Obsolete ("Use 'InputTopLeft' instead.")]
 		[CoreImageFilterProperty ("inputTopLeft")]
 		CIVector TopLeft { get; set; }
 
+		[Obsolete ("Use 'InputBottomLeft' instead.")]
 		[CoreImageFilterProperty ("inputBottomLeft")]
 		CIVector BottomLeft { get; set; }
+#endif
+
+		[CoreImageFilterProperty ("inputTopRight")]
+		CGPoint InputTopRight { get; set; }
+
+		[CoreImageFilterProperty ("inputBottomRight")]
+		CGPoint InputBottomRight { get; set; }
+
+		[CoreImageFilterProperty ("inputTopLeft")]
+		CGPoint InputTopLeft { get; set; }
+
+		[CoreImageFilterProperty ("inputBottomLeft")]
+		CGPoint InputBottomLeft { get; set; }
 	}
 
 	[CoreImageFilter]
 	[iOS (13,0)]
 	[TV (13,0)]
 	[Mac (10,15)]
 	[BaseType (typeof (CIKeystoneCorrection))]
-	interface CIKeystoneCorrectionCombined {
+	interface CIKeystoneCorrectionCombined : ICIKeystoneCorrectionCombinedProtocol {
+
+		[CoreImageFilterProperty ("outputTransform")]
+		CGAffineTransform OutputTransform { get; }
 	}
 
 	[CoreImageFilter]
 	[iOS (13,0)]
 	[TV (13,0)]
 	[Mac (10,15)]
 	[BaseType (typeof (CIKeystoneCorrection))]
-	interface CIKeystoneCorrectionHorizontal {
+	interface CIKeystoneCorrectionHorizontal : ICIKeystoneCorrectionHorizontalProtocol {
+
+#if false // no documentation about the type
```

Comment:
I thought we tried not to push dead code 😉. Couldn't it just be in the xtro ignore file with a comment?

Reply:
there's (almost) no headers for CIFilters, so no xtro reminders

_almost_ because some stuff is now defined in protocols - and those are in headers, but the rest is not (it's all key/value based)